lib/rack/cache/storage.rb

```diff
@@ -38,6 +38,9 @@ def create_store(type, uri)
         else
           fail "Unknown storage provider: #{uri.to_s}"
         end
+      elsif uri.nil?
+        # hack to support setting entitystore to nil, which doesn't persist request bodies
+        type.const_get('NIL').resolve(uri)
```

Comment:
call Dummy.new here ?
maybe set the uri to Dummy.new earlier / when initializing ?